crates/transaction-emitter-lib/src/emitter/submission_worker.rs

```diff
@@ -243,11 +248,21 @@ impl SubmissionWorker {
             )
             .await;
 
-        let (num_committed, num_expired) = update_seq_num_and_get_num_expired(
-            &mut self.accounts,
-            account_to_start_and_end_seq_num,
-            latest_fetched_counts,
-        );
+        // self.accounts.iter().for_each(|account| {})
+        for account in self.accounts.iter_mut() {
+            update_account_seq_num(
+                Arc::get_mut(account).unwrap(),
+                &account_to_start_and_end_seq_num,
+                &latest_fetched_counts,
+            );
+        }
+        let (num_committed, num_expired) =
+            count_committed_expired_stats(account_to_start_and_end_seq_num, latest_fetched_counts);
+        // let (num_committed, num_expired) = update_seq_num_and_get_num_expired(
```

Comment:
are these intentionally removed? should remove them?

Reply:
Fixed. `update_seq_num_and_get_num_expired()` got refactored away because I decided it was two operations awkwardly joined together with awkward value passing boundaries, which I think got more awkward when some of the values changed type due to other refactoring here. All cleaned up now.